compiler/passes/errorHandling.cpp

```diff
@@ -285,6 +285,9 @@ bool ErrorHandlingVisitor::enterCallExpr(CallExpr* node) {
 
       VarSymbol* errorVar    = NULL;
       BlockStmt* errorPolicy = new BlockStmt();
+      Expr*      insert      = node->getStmtExpr();
+      if (insert == NULL)
+        insert = node;
```

Comment:
I have no idea why `node->insertAtTail()` still works, but `node->insertBefore()` was switched to `insert->insertBefore()`.